### PR TITLE
add syntacx highliht

### PR DESCRIPTION
**Pull request by diced**

### CODE REVIEW

**Review by advilm — APPROVED** [review]

**Comment by rpxs** [comment]
no

**Comment by rpxs** [comment]
Messed up the indent

File: README.md [hunk 1]
```diff
@@ -30,41 +32,45 @@
...
       revealkey: true,
     });
-
+```
 **To Perform Searches**
 
 This package interacts directly with google's api. The base url can be retrieved by doing
-```const { apiurl } = require('ytsearcher');```
+```js
+const { apiurl } = require('ytsearcher');
```
Comment (wzhouwzhou) [inline README.md:56]: Would be careful about the indentation here

File: README.md [hunk 2]
```diff
@@ -30,41 +32,45 @@
...
     // You can customize your search with like so:
     let resultB = await searcher.search('Another Query', searchOptions)
-
+```
 A list of options is available [here](https://developers.google.com/youtube/v3/docs/search/list)
 
 Or you can fetch the list via:
-```const { validOptions } = require('ytsearcher');``` which will return the array.
+```js
+const { validOptions } = require('ytsearcher');
```
Comment (wzhouwzhou) [inline README.md:68]: Above comment; indentation applies here as well

**Review by wzhouwzhou — CHANGES_REQUESTED** [review]

**Comment by diced** [comment]
remember this commit fixes your dogwater md syntax

**Comment by rpxs** [comment]
who asked

**Comment by diced** [comment]
> who asked

you're wrong your mom asked

**Comment by rpxs** [comment]
ok you got me there

File: README.md [hunk 3]
```diff
@@ -122,37 +122,36 @@ which will return the array.
...
+const APIKEY = "12345"; // replace me
+const QUERY = "Anything you want"; // replace me too
 
-    ytsearcher.search(QUERY, { type: 'video' })
-    .then(searchResult => {
+const { YTSearcher } = require('ytsearcher');
+const ytsearcher = new YTSearcher(APIKEY);
 
-      searchResult.nextPage()
+ytsearcher.search(QUERY, { type: 'video' })
```
Comment (wzhouwzhou) [inline README.md]: Let's put the `.then`'s on the same line for style purposes

**Review by wzhouwzhou — CHANGES_REQUESTED** [review]

**Comment by diced** [comment]
mad bc u bad

**Review by rpxs — CHANGES_REQUESTED** [review]
Fix indents, all good.

**Comment by diced** [comment]
ok is this good

**Review by rpxs — APPROVED** [review]

**Comment by diced** [comment]
U Snooze U Lose

**Review by wzhouwzhou — CHANGES_REQUESTED** [review]
Let's also change over the badge to be latest, and update references to 2.5 KB to now be 2.2 KB as per https://bundlephobia.com/result?p=ytsearcher@1.2.4

**Comment by diced** [comment]
god you are so needy

**Review by wzhouwzhou — APPROVED** [review]
LGTM now!

**Review by Vetlix — APPROVED** [review]
merge!!

**Review by rpxs — COMMENTED** [review]
pogger commit

**Review by rpxs — APPROVED** [review]

**Comment by rpxs** [comment]
pressed wrong button so i only review instead of approve but it works now

**Comment by diced** [comment]
@wzhouwzhou merg

**Review by tbnritzdoge — APPROVED** [review]
these changes are critical to the future of the NODE.JAVASCRIPT developr enviorment and must be merged with no delay!!

**Review by rpxs — APPROVED** [review]
didnt end up eating taco bell today ☹️ ☹️ instead had mango juice 😃

**Comment by rpxs** [comment]
poggers!

**Comment by advilm** [comment]
mango juice op

**Pull request merged**